opensrp-chw-referral/src/main/java/org/smartregister/chw/referral/model/BaseReferralRegisterModel.java

```diff
@@ -0,0 +1,45 @@
+package org.smartregister.chw.referral.model;
+
+import org.json.JSONObject;
+import org.smartregister.chw.referral.contract.BaseReferralRegisterContract;
+import org.smartregister.chw.referral.util.JsonFormUtils;
+
+import java.util.List;
+
+public class BaseReferralRegisterModel implements BaseReferralRegisterContract.Model {
+    @Override
+    public void registerViewConfigurations(List<String> viewIdentifiers) {
+//        implement
+
+    }
+
+    @Override
+    public void unregisterViewConfiguration(List<String> viewIdentifiers) {
+//        implement
+
+    }
+
+    @Override
+    public void saveLanguage(String language) {
+//        implement
+
+    }
+
+    @Override
+    public String getLocationId(String locationName) {
+        return null;
+    }
+
+    @Override
+    public JSONObject getFormAsJson(String formName, String entityId, String currentLocationId) throws Exception {
+        JSONObject jsonObject = JsonFormUtils.getFormAsJson(formName);
+        JsonFormUtils.getRegistrationForm(jsonObject, entityId, currentLocationId);
```

Comment:
```JsonFormUtils.getRegistrationForm``` method can be renamed better, naming it as a getter method is confusing since it is not returning any value. Or you might consider making it return a value instead.

Reply:
@ellykits Could you provide a better suggestion in mind for this?

Reply:
You can possibly name it as ```addFormMetadata```